packages/blockchain-wallet-v4-frontend/src/data/components/buySell/sagas.ts

```diff
@@ -595,8 +597,12 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
 
   const orderConfirmCheck = function* (orderId) {
     const order: ReturnType<typeof api.getBSOrder> = yield call(api.getBSOrder, orderId)
-
-    if (order.state === 'FINISHED' || order.state === 'FAILED' || order.state === 'CANCELED') {
+    if (
+      order.state === 'FINISHED' ||
+      order.state === 'FAILED' ||
+      order.state === 'CANCELED' ||
```

Comment:
I've gone back and fourth on converting everything to enums. In this case I think it's actually better to be a union type of strings because you get all the same benefits of an enum without the extra keystrokes. Also this PR would grow quite a bit if I made that change and it's already a pretty decent size. Checkout the thread I started about this in slack, and let's discuss.